biscuit-capi/src/lib.rs

```diff
@@ -0,0 +1,1251 @@
+use rand::prelude::*;
```

Comment:
shouldn't this remove the file `biscuit-auth/src/capi.rs`?

Reply:
Yes, it's just a copy paste from `biscuit-auth/src/capi.rs` and I just moved into the new crate. That's why I changed some visibility in some modules

Reply:
could you delete `biscuit-auth/src/capi.rs` then? That way git will detect that the file moved and we'll be able to see the actual changes in the file

Reply:
done sorry 🙇